### PR TITLE
Freemabd1/rw 124 added icd10 criteria tree and updated liquibase build run local version of trees

### PR DESCRIPTION
**Pull request by freemabd1**
**We now have 2 activities for the cdr liquibase plugin:**
```
liquibase {
    activities {
        main {
            changeLogFile "changelog/db.changelog-master.xml"
            url "jdbc:mysql://${db_host}:${db_port}/cdr"
            username "liquibase"
            password "${liquibase_password}"
        }
        local {
            changeLogFile "changelog-local/db.changelog-master.xml"
            url "jdbc:mysql://${db_host}:${db_port}/cdr"
            username "liquibase"
            password "${liquibase_password}"
        }
        runList = project.ext.runList
    }
}
```

**in the run-migrations.sh for local deployments we now call the update task in liquibase with the specific activity name like so:**
```
echo "Upgrading database..."
../gradlew update -PrunList=local
```

**For deployment to test env we specify the main activity like so (devstart.rb):**
`ctx.common.run_inline("#{ctx.gradlew_path} --info update -PrunList=main")`

### CODE REVIEW

File: api/db-cdr/changelog-local/db.changelog-5.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="brianfreeman" id="changelog-5" runAlways="true">
+
+    <validCheckSum>ANY</validCheckSum>
+
+    <loadData tableName="criteria" file="csv/icd10_criteria-local.csv" encoding="UTF-8" quotchar='"'>
```
Comment (danqrodney) [inline api/db-cdr/changelog-local/db.changelog-5.xml]: While you're at it, should we make a smaller version of ICD 9 so local builds go faster?

Comment (freemabd1) [reply api/db-cdr/changelog-local/db.changelog-5.xml]: icd9 file is about the same size as the subset icd10 file. It's pretty fast currently(2.6s).

changelog-1::brianfreeman ran successfully in 14ms
changelog-2::brianfreeman: Table criteria created
changelog-2::brianfreeman: Index idx_type_parent_id created
changelog-2::brianfreeman: ChangeSet changelog/db.changelog-2.xml::changelog-2::brianfreeman ran successfully in 27ms
changelog-3::brianfreeman: Data loaded from csv/icd9_criteria.csv into criteria
changelog-3::brianfreeman: ChangeSet changelog/db.changelog-3.xml::changelog-3::brianfreeman ran successfully in 2614ms
changelog-4::brianfreeman: Data loaded from csv/demo_criteria.csv into criteria
changelog-4::brianfreeman: ChangeSet changelog/db.changelog-4.xml::changelog-4::brianfreeman ran successfully in 7ms
changelog-5::brianfreeman: Data loaded from csv/icd10_criteria-local.csv into criteria
changelog-5::brianfreeman: ChangeSet changelog-local/db.changelog-5.xml::changelog-5::brianfreeman ran successfully in 1848ms

Comment (freemabd1) [reply api/db-cdr/changelog-local/db.changelog-5.xml]: And I will add comments to README

Comment (freemabd1) [reply api/db-cdr/changelog-local/db.changelog-5.xml]: I updated the the README at the very bottom under API Server Database Updates.

**Pull request merged**